data/json/materials.json

```diff
@@ -1551,6 +1551,24 @@
     "bash_dmg_verb": "damaged",
     "cut_dmg_verb": "damaged"
   },
+  {
+    "type": "material",
+    "id": "mercury",
+    "name": "Mercury",
+    "density": 13.53,
+    "specific_heat_liquid": 0.1395,
+    "specific_heat_solid": 0.1395,
+    "bash_resist": 0,
```

Comment:
Also when frozen wouldn't it have resists against bash, cut and bullet?

Reply:
I've based those stats on lead. It has exactly the same damage resistance stats, and solid mercury would presumably have even less as it appears that solid mercury can be cut even with knife.